profiler/src/profile_max_pool2d_fwd.cpp

```diff
@@ -49,9 +49,19 @@ struct maxPoolFwdArgParser
     }
 };
 
+enum struct PoolDataType
+{
+    F32,    // 0
```

Comment:
[Consider] To avoid this comment `//0 //1...,` we can either assign 0 to the first element` (F32 = 0)` or delete the comment (by default it's known)

Reply:
Assigning first value to 0 seems much clearer that the current solutions, thx!